src/utilities/device_utils.c

```diff
@@ -2188,7 +2187,7 @@ hypre_CudaCompileFlagCheck()
 /*--------------------------------------------------------------------
  * hypreGPUKernel_DiagScaleVector
  *
- * Assumes vectors components are stored contiguously.
+ * Assumes vectors components are stored columwise.
```

Comment:
Neither of these comments makes sense to me.  Is it even necessary?  This is a multivector, and the way multivector data is stored only needs to be documented once in an appropriate header file.

Reply:
Yes, agreed!